iroh-net/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 //! Peer-to-peer QUIC connections.
 //!
+//! <div class="warning">
+//!
+//! The `iroh-net` crate has been renamed to `iroh`.  Please use the
+//! `iroh` crate instead.
+//!
+//! </div>
+//!
```

Comment:
isn't there sth like `#![deprecated]` that we can do?

Reply:
Flub answered this question here: https://github.com/n0-computer/iroh/pull/3116#issuecomment-2582743069

Reply:
I think we should do it, otherwise I think this will be very easily missed by downstream users, you can always use `[allow(deprectated)]`